### PR TITLE
Character select screen highlighting using keyboard

### PR DESCRIPTION
**Pull request by miishin**
Added a highlighting cursor. 

Still need to fix:

- Pressing spacebar adds units to the queue correctly, but does not toggle the actual button on/off, which is weird because I call the same function that is called when clicking with a mouse (which works). 
- The cursor disappears under tiles that have been clicked (toggled on), which also makes no sense because the highlight sprite is on a higher layer.

### CODE REVIEW

File: assets/scripts/ability.gd [hunk 1]
```diff
@@ -36,9 +36,16 @@ var debuff : int
 # Size of AoE of ability (AoE = 1 for singular target) in tiles
 var aoe : int
 
-# Accuracy (% to hit )
-var accuracy : int
```
Comment (IsaacPD) [inline assets/scripts/ability.gd:40]: Please rebase onto main, or remove this commit that removes accuracy since we already have these changes on main

**Review by IsaacPD — CHANGES_REQUESTED** [review]

File: assets/scripts/character_select.gd [hunk 2]
```diff
@@ -59,23 +59,58 @@ onready var button_map = {
 	"c10" : $GridContainer/c10
 }
 
+var all_buttons : Array
+var button_grid : Array
+var current_selection : Vector2
+
 func _ready():
+	all_buttons = $GridContainer.get_children()
+	button_grid = [all_buttons.slice(0, 4), all_buttons.slice(5, 9), [$back, $forward]]
```
[the remark this answers is not in the document]
Comment (miishin) [reply assets/scripts/character_select.gd:68]: latest push fixed it. I forgot to bind back/forward in the keyboard selection lol

File: debug.log [hunk 3]
```diff
@@ -0,0 +1 @@
+[0112/213931.920:ERROR:settings.cc(325)] Settings version is not 1
```
[the remark this answers is not in the document]
Comment (miishin) [reply debug.log:1]: no idea

[the remark this answers is not in the document]
Comment (IsaacPD) [reply debug.log:1]: can you just delete this file please -_-

[the remark this answers is not in the document]
Comment (miishin) [reply debug.log]: deleted >:)

**Review by IsaacPD — APPROVED** [review]

**Pull request merged**